crates/pathfinder/src/cairo/ext_py.rs

```diff
@@ -520,6 +537,110 @@ mod tests {
         jh.await.unwrap();
     }
 
+    #[test_log::test(tokio::test)]
+    #[ignore]
```

Comment:
Or however `ignore` text works.
```suggestion
    #[ignore("Requires python env")]
```

Reply:
Done in next push.

Reply:
It was `#[ignore = "reason"]`, gets output after comma:

```
test cairo::ext_py::tests::call_like_in_python_ten_times ... ignored, needs python venv
```